### PR TITLE
Create 3-0-implementation-plan.md

### PR DESCRIPTION
**Pull request by avuim**
Initial proposal of MVP backlog for implementation stages into committed  SSP/DSPs

### CODE REVIEW

**Comment by avuim** [comment]
Protocol of our call from 12th Sep 2022

File: 3-0-implementation-plan.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Implementation Plan
+
+The next stage after specifying a potential approach for publisher curated addressability is a MVP implementation with relevant vendors (SSPs and DSPs).
+In the following we describe an implementation plan for involved parties, creating a MVP.
+
+### Package I
+
+* Adherence of (SSP to DSP) "authorized connections" restriction configuration
```
Comment (asr-enid) [inline 3-0-implementation-plan.md:8]: For the sake of completeness 

"Adherence to (Publisher to SSP, SSP to DSP) "authorized connections" restriction configuration

File: 3-0-implementation-plan.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Adherence of DSP "seat level" restriction configuration but without "authorized feature" binding
```
Comment (asr-enid) [inline 3-0-implementation-plan.md:9]: Need to cycle back on this not sure what we mean by this

File: 3-0-implementation-plan.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* No restriction of internal DSP graph usage (e.g. Adform Fusion ID, VirtualMinds Any ID)
```
Comment (asr-enid) [inline 3-0-implementation-plan.md:10]: My understanding was that Graph Usage (yes/no generally not seat specific) per ID is a must to be configurable (and is possible already)

File: 3-0-implementation-plan.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* but no ID exposure, since ID export per default deactivated
```
Comment (asr-enid) [inline 3-0-implementation-plan.md:11]: See above I think both is part of I and also possible

File: 3-0-implementation-plan.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+* PCA config not automatically crawled yet
+
+### Package II
+
+* Partially adherence of "authorized feature" config especially in respect of IDs being used in DSP internal graphs or not
```
Comment (asr-enid) [inline 3-0-implementation-plan.md:16]: See above

**Review by asr-enid — CHANGES_REQUESTED** [review]

File: 3-0-implementation-plan.md (hunk 3 above)
Comment (avuim) [reply 3-0-implementation-plan.md:10]: @jsr-adf clearly statet, that the internal usage (e.g. Fusion ID Graph) cannot be restricted in the first step but not external exposure of ID takes place, therefore "could" be accepted in combination with contractual agreement. Jochen, maybe you comment on this.

File: 3-0-implementation-plan.md (hunk 1 above)
Comment (avuim) [reply 3-0-implementation-plan.md:8]: yeah, dont't mind, but Pub to SSP/bidder ist anyway possible with prebid config by now, therefore no implementation requirement in my opinion.

File: 3-0-implementation-plan.md (hunk 2 above)
Comment (avuim) [reply 3-0-implementation-plan.md:9]: Complying with config for SSP to DSP signalling down to seat level but without the detail individual control of features on seat level, but rather globally per DSP (e.g. no raw ID export)

File: 3-0-implementation-plan.md (hunk 4 above)
Comment (avuim) [reply 3-0-implementation-plan.md:11]: Let's discuss in next call

File: 3-0-implementation-plan.md (hunk 1 above)
Comment (halthauser) [reply 3-0-implementation-plan.md:8]: Ok with [avuim](https://github.com/avuim)

File: 3-0-implementation-plan.md (hunk 2 above)
Comment (halthauser) [reply 3-0-implementation-plan.md:9]: Ok with [avuim](https://github.com/avuim)

File: 3-0-implementation-plan.md (hunk 3 above)
Comment (halthauser) [reply 3-0-implementation-plan.md:10]: Also on the same page w/ [avuim](https://github.com/avuim), can't be restricted in the first step

File: 3-0-implementation-plan.md (hunk 4 above)
Comment (halthauser) [reply 3-0-implementation-plan.md:11]: Like discussed okay for us.

**Pull request merged**